lib/ex_aws/sqs/parsers.ex

```diff
@@ -119,11 +119,17 @@ if Code.ensure_loaded?(SweetXml) do
                           ]
                         ])
 
-      fixed_attributes = parsed_body
-      |> fix_attributes([:message, :attributes], &attribute_list_to_map/1)
-      |> fix_attributes([:message, :message_attributes], &message_attributes_to_map/1)
+      new_messages = parsed_body
+      |> Map.get(:messages)
```

Comment:
Will do, but it looks like sweetXML returns an empty list if messages isn't present (There's a test for this case).
I'll still add it as it's clearer.